package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Web Version",
   "scripts": {
     "build": "cross-env NODE_ENV=production webpack",
-    "dev": "cross-env NODE_ENV=development webpack serve --inline --open --mode=development",
```

Comment:
Removing `open` option since often devs have a browser already open when they restart the server. Also `--mode` here is redundant since we are setting `NODE_ENV`